src/server.js

```diff
@@ -0,0 +1,33 @@
+const app = require('./app');
+const http = require('http');
+const mongoose = require('mongoose');
+
+const uri = ('mongodb://localhost:27017/test' || process.env.mongoSecret);
+mongoose.connect(uri, {useNewUrlParser: true, useFindAndModify: false})
+.then(() => {
+})
+.catch((err) => {
+  console.log(err);
```

Comment:
Should probably use `console.error()` instead, so it goes to stderr instead of stdout

Reply:
Updated.